src/main/java/com/dougnoel/sentinel/elements/Element.java

```diff
@@ -745,6 +746,37 @@ public boolean classContains(String value) {
 		return false;
 	}
 
+	/**
+	 * Clicks a point relative to the center of the element.
+	 * To click the top right corner of an element of a width of 20 and a height of 10, you would need to pass Point(10, 5)
+	 * @param point The offset from the center of the element to click on
+	 * @return Element (for chaining)
+	 */
```

Comment:
This is the new W3 standard in the actions library. The center of the element is a cartesian 0,0. I was shocked because I used to do it in the upper left as well. 
Note: https://github.com/mozilla/geckodriver/issues/789
Webdriver spec: https://w3c.github.io/webdriver/webdriver-spec.html#dfn-dispatch-a-pointermove-action